ote/src/cljs/ote/app/controller/admin.cljs

```diff
@@ -147,18 +146,19 @@
 
   SearchUsers
   (process-event [_ app]
-    (comm/get! (str "admin/user?type=any&search=" (get-in app [:admin :user-listing :user-filter]))
-                {:on-success (tuck/send-async! ->SearchUsersResponse)
-                 :on-failure (tuck/send-async! ->SearchUsersResponse)})
+    (let [filter (get-in app [:admin :user-listing :user-filter])]
+      (comm/get! (str "admin/user"
+                      (when filter (str "?type=any&search=" filter)))
+                 {:on-success (tuck/send-async! ->SearchUsersResponse)
+                  :on-failure (tuck/send-async! ->SearchUsersResponse)}))
     (assoc-in app [:admin :user-listing :loading?] true))
 
   ConfirmDeleteUser
   (process-event [{id :id} app]
     (if (= id (:ensured-id (get-user-by-id app id)))
       (comm/delete! (str "admin/user/" id)
                     nil
-                    {:on-success (tuck/send-async! ->ConfirmDeleteUserResponse)
-                     :on-failure (tuck/send-async! ->ConfirmDeleteUserResponseFailure)})
+                    {:on-success (tuck/send-async! ->ConfirmDeleteUserResponse)})
```

Comment:
Why not on-failure case?

Reply:
During development I added extra on-failures and this was removed by accident while removing those because it seemed to need proper handling. But it's better to keep the old functionality, no time to refactor everything :)

Reply:
So on-failure restored, thanks!